File: Dev/Code/Community.Entity/Model/ServiceOrder/ServiceOrderEntity.cs
using System;
using System.Collections.Generic;
using Community.Entity.Model.Member;
using Community.Entity.Model.MemberAddress;
using Community.Entity.Model.Order;
using Community.Entity.Model.ServiceOrderDetail;
using YooPoon.Core.Data;

namespace Community.Entity.Model.ServiceOrder
{
	public class ServiceOrderEntity : IBaseEntity
	{
		/// <summary>
		/// Id
		/// </summary>
		public virtual int Id { get; set; }
		/// <summary>
		/// 订单号
		/// </summary>
		public virtual string OrderNo { get; set; }
		/// <summary>
		/// 添加时间
		/// </summary>
		public virtual DateTime AddTime { get; set; }

        /// <summary>
        /// 更新人
        /// </summary>
        public virtual int UpdUser { get; set; }
        /// <summary>
        /// 更新时间
        /// </summary>
        public virtual DateTime UpdTime { get; set; }

		/// <summary>
		/// 添加人
		/// </summary>
		public virtual int AddUser { get; set; }
		/// <summary>
		/// 费用
		/// </summary>
		public virtual decimal Flee { get; set; }
//		/// <summary>
//		/// 地址
//		/// </summary>
//		public virtual string Address { get; set; }
		/// <summary>
		/// 服务时间
		/// </summary>
		public virtual DateTime Servicetime { get; set; }
		/// <summary>
		/// 备注
		/// </summary>
		public virtual string Remark { get; set; }

        /// <summary>
        /// 状态
        /// </summary>
        public virtual EnumOrderStatus Status { get; set; }

        /// <summary>
        /// 订单明细
        /// </summary>
        public virtual List<ServiceOrderDetailEntity> Details { get; set; }

        /// <summary>
        /// 下单人实体
        /// </summary>
        public virtual MemberEntity AddMember { get; set; }

        /// <summary>
        /// 地址
        /// </summary>
        public virtual MemberAddressEntity Address { get; set; }
	}
}

File: Dev/Android/YoopoonMarket/src/com/yoopoon/market/domain/ServiceOrderEntity.java
package com.yoopoon.market.domain;

import java.io.Serializable;
import java.util.List;

public class ServiceOrderEntity implements Serializable {

	/**
	 * @fieldName: serialVersionUID
	 * @fieldType: long
	 * @Description: TODO
	 */
	private static final long serialVersionUID = 1L;

	public int Id;
	public String OrderNo;
	public String Addtime;
	public int AddUser;
	public int UpdUser;
	public String UpdTime;
	public float Flee;
	public String Address;
	public String Servicetime;
	public String Remark;
	public int Status;
	public String StatusString;
	public String UserName;
	public int MemberAddressId;
	public List<ServiceOrderDetail> Details;

	@Override
	public String toString() {
		return "ServiceOrderEntity [Id=" + Id + ", OrderNo=" + OrderNo + ", Addtime=" + Addtime + ", AddUser="
				+ AddUser + ", UpdUser=" + UpdUser + ", UpdTime=" + UpdTime + ", Flee=" + Flee + ", Address=" + Address
				+ ", Servicetime=" + Servicetime + ", Remark=" + Remark + ", Status=" + Status + ", StatusString="
				+ StatusString + ", UserName=" + UserName + ", MemberAddressId=" + MemberAddressId + ", Details="
				+ Details + "]";
	}

}
